op-guide/binary-deployment.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Deploy TiDB Using the Binary
+category: operations
+---
+
+# Deploy TiDB Using the Binary
+
+This guide provides installation instructions from tarball on Linux. A complete TiDB cluster contains PD, TiKV, and TiDB. To start the database service, follow the order of PD -> TiKV -> TiDB. To stop the database service, follow the order of stopping TiDB -> TiKV -> PD.
+
+This document describes the binary deployment of three scenarios:
+
+1. [Single node cluster deployment](#single-node-cluster-deployment) for trying out TiDB.
+2. [Multiple nodes cluster deployment for testing](#multiple-nodes-cluster-deployment-for-test) TiDB across multiple nodes and exploring features in more detail.
+3. [Multiple nodes cluster deployment](#multiple-nodes-cluster-deployment) for production deployments.
+
+## Prepare
+
+Before you start, see [TiDB architecture](/overview.md#tidb-architecture) and [Software and Hardware Requirements](/op-guide/recommendation.md). Make sure the following requirements are satisfied:
+
+### Operating system
+
+For the operating system, it is recommended to use RHEL/CentOS 7.3 or higher. The following additional requirements are recommended:
+
+| Configuration | Description |
+| :-- | :-------------------- |
+| Supported Platform | RHEL/CentOS 7.3+ ([more details](/op-guide/recommendation.md)) |
+| File System  |  ext4 is recommended |
+| Swap Space  |  Should be disabled  |
+| Disk Block Size  |  Set the system disk `Block` size to `4096` |
+
+### Network and firewall
+
+| Configuration | Description |
+| :-- | :------------------- |
+| Firewall/Port | Check whether the ports required by TiDB are accessible between the nodes |
+
+### Operating system parameters
+
+| Configuration | Description |
+| :-- | :-------------------------- |
+| Nice Limits | For system users, set the default value of `nice` in TiDB to `0` |
+| min_free_kbytes | The setting for `vm.min_free_kbytes` in `sysctl.conf` needs to be high enough |
+| User Open Files Limit | For database administrators, set the number of TiDB open files to `1000000` |
+| System Open File Limits | Set the number of system open files to `1000000` |
+| User Process Limits | For TiDB users, set the `nproc` value to `4096` in `limits.conf` |
+| Address Space Limits | For TiDB users, set the space to `unlimited` in `limits.conf` |
+| File Size Limits | For TiDB users, set the `fsize` value to `unlimited` in `limits.conf` |
+| Disk Readahead | Set the value of the `readahead` data disk to `4096` at a minimum |
+| NTP service | Configure the NTP time synchronization service for each node |
+| SELinux  | Turn off the SELinux service for each node |
+| CPU Frequency Scaling | It is recommended to turn on CPU overclocking |
+| Transparent Hugepages | For Red Hat 7+ and CentOS 7+ systems, it is required to set the Transparent Hugepages to `always` |
+| I/O Scheduler | Set the I/O Scheduler of data disks to the `deadline` mode |
+| vm.swappiness | Set `vm.swappiness = 0` |
```

Comment:
In order to be consistent with the parameters in tidb-ansible, it is recommended to add the following two parameters.

| vm.swappiness | Set `net.core.somaxconn = 32768` in `sysctl.conf` |
| net.ipv4.tcp_syncookies | Set `net.ipv4.tcp_syncookies = 0` in `sysctl.conf` |